core/src/main/java/com/airbnb/aerosolve/core/transforms/DayOfMonthTransform.java

```diff
@@ -0,0 +1,15 @@
+package com.airbnb.aerosolve.core.transforms;
+
+
+import java.util.Calendar;
+
+/**
+ * Get day of month from date
+ */
+public class DayOfMonthTransform extends DateTransform {
+  @Override
+  public Double doDateTransform(Calendar cal) {
+    // First day of month is 1
+    return (double)cal.get(Calendar.DAY_OF_MONTH);
```

Comment:
instead of creating new class, how about a config, field2,  = day_of_month,  since in  DateDiffTransform , its store date, maybe we just need one class?